SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -594,6 +594,14 @@ SPATIALGDKEDITOR_API bool GeneratedSchemaFolderExists()
 	return PlatformFile.DirectoryExists(*SchemaOutputPath);
 }
 
+SPATIALGDKEDITOR_API bool GeneratedSchemaDatabaseExists()
+{
+	const FString SchemaDatabasePackagePath = FPaths::Combine(FPaths::ProjectContentDir(), TEXT("Spatial/SchemaDatabase"));
```

Comment:
Should "Spatial/SchemaDatabase" part be a const somewhere? I can imagine it being used in different places across the source.

Reply:
Could go in https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h if you like.

Reply:
Yeah, there's actually another reference to it in that same file so it would make sense. Would it be better to include those changes in this PR or do a separate one though? Might be worth scouring the code to see if there are any other hard-coded paths to tidy up at the same time.

Reply:
This PR's small enough that doing it here won't confuse things too much.

Reply:
Alright, I'll jump on that in a bit then.

Reply:
I've gone through and replaced the various path references with new constants. I ended up setting two different constants for file and asset paths as I'm not aware of an easy way to convert between them.